variants/lionbits3/pins_arduino.h

```diff
@@ -0,0 +1,109 @@
+static const uint8_t A2  = 3;   //I/O,  GPIO3,    TOUCH3, ADC1_CH2, */MAX 5V,   U0RXD_in,U0CTS_in,U0DSR_in,U0TXD_out,UoRTS_out,U1RXD_in,U1CTS_in,U1DSR_in,U1TXD_out,U1RTS_out,U1DTR_out,U2RXD_in,U2CTS_in,U2DSR_in,U2TXD_out,U2RTS_out,U2DTR_out,LEDPWM 
+static const uint8_t A3  = 4;   //I/O,  GPIO4,    TOUCH4, ADC1_CH3, */MAX 5V,   U0RXD_in,U0CTS_in,U0DSR_in,U0TXD_out,UoRTS_out,U1RXD_in,U1CTS_in,U1DSR_in,U1TXD_out,U1RTS_out,U1DTR_out,U2RXD_in,U2CTS_in,U2DSR_in,U2TXD_out,U2RTS_out,U2DTR_out,LEDPWM
+static const uint8_t A4  = 5;   //I/O,  GPIO5,    TOUCH5, ADC1_CH4, */MAX 5V,   U0RXD_in,U0CTS_in,U0DSR_in,U0TXD_out,UoRTS_out,U1RXD_in,U1CTS_in,U1DSR_in,U1TXD_out,U1RTS_out,U1DTR_out,U2RXD_in,U2CTS_in,U2DSR_in,U2TXD_out,U2RTS_out,U2DTR_out,LEDPWM  
+static const uint8_t A5  = 6;   //I/O,  GPIO6,    TOUCH6, ADC1_CH5, */MAX 5V,   U0RXD_in,U0CTS_in,U0DSR_in,U0TXD_out,UoRTS_out,U1RXD_in,U1CTS_in,U1DSR_in,U1TXD_out,U1RTS_out,U1DTR_out,U2RXD_in,U2CTS_in,U2DSR_in,U2TXD_out,U2RTS_out,U2DTR_out,LEDPWM
+static const uint8_t AD1 = 7;   //I/O,  GPIO7,    TOUCH6, ADC1_CH5, */MAX 5V,   U0RXD_in,U0CTS_in,U0DSR_in,U0TXD_out,UoRTS_out,U1RXD_in,U1CTS_in,U1DSR_in,U1TXD_out,U1RTS_out,U1DTR_out,U2RXD_in,U2CTS_in,U2DSR_in,U2TXD_out,U2RTS_out,U2DTR_out,LEDPWM
+
+// Inbuilt Display Unit 128*128 ST7735 Driver New
+
+static const uint8_t SDA = 40; // GPIO40;
+static const uint8_t SCL = 41; // GPIO41;
+
+/* Hardware HSPI */
+static const uint8_t MOSI = 35; // GPIO35;
+static const uint8_t MISO = 37; // GPIO37;
+static const uint8_t SCK = 36;  // GPIO36;
+static const uint8_t SS = 34;   // GPIO34;
+static const uint8_t SDO  = 35; // GPIO35;
+static const uint8_t SDI  = 37; // GPIO37;
+//---------------------------------- 
+
+static const uint8_t TFT_RST = 38;  //GPIO38;      
+static const uint8_t TFT_SCLK = 35;  //GPIO35;     
+static const uint8_t TFT_CS = 42;   //GPIO42;       
+static const uint8_t TFT_DC = 37;   //GPIO37;  
+static const uint8_t TFT_MOSI = 36;  //GPIO36; 
+
+static const uint8_t LCD_A0 = 37;
+static const uint8_t LCD_BACK_LIGHT = 45;
+static const uint8_t DAC1 = 21; //  GPIO21, 
+//LCD aditional pins
+
+//Adafruit 128*128 ST7735 Driver New
+static const uint8_t rst = 38; 
+static const uint8_t sclk = 35; 
+static const uint8_t cs = 42; 
+static const uint8_t dc = 37; 
+static const uint8_t mosi = 36; 
+
+//--------------------
+//Digital to Analog Converter 
+#define DA1               (25)  //I/O GPIO25, DAC_1, ADC2_CH8, RTC_GPIO6, EMAC_RXD0
+#define DA2               (26)  //I/O GPIO26, DAC_2, ADC2_CH9, RTC_GPIO7, EMAC_RXD1
+
+//-------------------------------------------------------------------
+
+ #define VP 36  //  GPIO36, ADC1_CH0, RTC_GPIO0
+ #define VN 39  //  GPIO39, ADC1_CH3, RTC_GPIO3
+
+
+//-------------------------------------------------------------------
+
+ 
+
```

Comment:
Remove this empty part.
```suggestion
//-------------------------------------------------------------------
```